src/_premake_init.lua

```diff
@@ -241,6 +241,24 @@
 	}
 
 	api.register {
+		name = "debugger",
+		scope = "config",
+		kind = "string",
+		allowed = {
+			"VisualStudio",
+			"GDB",
+			"LLDB",
+		}
+	}
```

Comment:
I wondered if this should be called 'debugtool' to match the other 2 new options below, but it didn't feel right.
I don't think "VisualStudio" is a 'debugtool'. That terminology just felt a bit awkward to me.